pod/favorite/static/js/video-favorites-card-delete.js

```diff
@@ -0,0 +1,29 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const cards = document.getElementsByClassName("draggable-container");
+    const title = document.getElementById("video_count")
+    for (let card of cards) {
+        const form = card.querySelector(".favorite-button-form-card");
+        form.addEventListener('submit', function (e) {
+            e.preventDefault();
+            const formData = new FormData(form);
+            fetch(form.action, {
+                method: form.method,
+                body: formData
+            })
+                .then(response => response.text()) // We take the HTML content of the response
+                .then(data => {
+                    const parser = new DOMParser();
+                    const html = parser.parseFromString(data, 'text/html');
+                    card.remove();
+                    const title = document.getElementById("video_count");
+                    title.replaceWith(html.getElementById("video_count"));
+                    document.title = html.title;
+                    addEventForReorganizedButton();
+                })
+                .catch(error => {
+                    if (!(error instanceof TypeError))
+                        alert(gettext('The deletion couldn\'t be completed...'));
```

Comment:
Je pense que le texte n'est pas assez clair : on a l'impression q'il a voulu supprimer la vidéo. Il faudrait un texte du genre "The video could not be removed from favorites..."